### ISSUE TITLE
SchemaBuilder toSql() returns `Sql[]`, not `Sql`.

### ISSUE THREAD

**Issue by pocketarc** [knex/knex#5593]
# Environment

Knex version: 2.4.2
Database + version: sqlite dialet (no connection, so no version)
OS: macOS 13 Ventura

# Bug

Calling toSQL() when building a schema returns an array of queries to execute, but the TypeScript definition has it as just a single object.

The type should be changed from `Sql` to `Sql[]`, to account for that.

```
const test = knex({
    client: "sqlite",
    useNullAsDefault: true,
});

let sql = test.schema.createTable("test", (table) => {
    table.string("name").notNullable();
}).toSQL();

// sql is an array with one `Sql` type object

**Comment by OlivierCavadenti** [comment]
Fixed in [#5594](https://github.com/knex/knex/pull/5594)

**Comment by OlivierCavadenti** [comment]
Released in [2.5.0](https://github.com/knex/knex/releases/tag/2.5.0)
